gems/smithy-cbor/lib/smithy-cbor/codec.rb

```diff
@@ -6,8 +6,6 @@
 module Smithy
   module CBOR
     # Codec that serializes and deserializes in CBOR format.
-    # TODO:
-    #   * Update implementation to handle event streams
```

Comment:
Do we not support eventstreams for CBOR?

Reply:
This TODO would linger a while. Also I think it wouldn't be implemented here? This would make the body of the event stream.

Reply:
That's fair - when the core eventstream components gets implemented, we prob do a circle around in all protocols